src/file_io.py

```diff
@@ -35,10 +39,11 @@ def save_project(file, plot_settings, datadict, datadict_clipboard,
 
 def read_project(file):
     project = pickle.loads(_read_file(file, None))
+    project_items = ["plot_settings", "data", "clipboard", "version"]
+    project.extend({key, None for key in project_items if key not in project})
```

Comment:
```suggestion
    project.extend({key: None for key in project_items if key not in project})
```
Oops, my bad

Reply:
Yep, didn't notice it immediately either, had it already fixed in my local version.